rpcd/etc/openstack_deploy/user_apt_packages.yml

```diff
@@ -0,0 +1,11 @@
+# This file will contain any apt packages that the deployer would like to pin to a particular version.
```

Comment:
It's created by this PR, see https://github.com/stevelle/rpc-extras/commit/e6eb0d1bc09d2d18f63ad4bcd7e95af42ca8ef9e